index.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const { getAllMovies, getByTitleOrDirector, saveNewMovie } = require('./controllers/movies')
+
+const app = express()
+
+app.get('/movies', getAllMovies)
+
+app.get('/movies/:input', getByTitleOrDirector)
+
+app.post('/', bodyParser.json(), saveNewMovie)
```

Comment:
This is not posting the movie to the correct location so it doesnt work. 

`app.post('/movies', bodyParser.json(), saveNewMovie)`